webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/nodes/NodeInfoService.scala

```diff
@@ -331,45 +331,39 @@ trait NodeInfoServiceCached extends NodeInfoService with NamedZioLogger with Cac
       }
 
       def getUpdatedCache(allNodeEntries: AllNodeEntries): IOResult[LocalNodeInfoCache] = {
-        //some map of things - mutable, yes
-        val nodes = MutMap[String, LDAPEntry]() //node_uuid -> entry
-        val nodeInventories = MutMap[String, LDAPEntry]() // node_uuid -> entry
-        val machineInventories = MutMap[String, LDAPEntry]() // machine_dn -> entry
-
         val t0 = System.currentTimeMillis
 
         // two vars to keep track of the new last modification time and entries csn
         var lastModif = lastKnowModification
         val entriesCSN = scala.collection.mutable.Buffer[String]()
 
-
-        //look for the maxed timestamp
-        (allNodeEntries.deleted ++ allNodeEntries.active).foreach { e =>
```

Comment:
the ++ is fairly expensive, as allNodeEntries.active is some pretty big object. We don't need to duplicate it